components/dashboards-web-component/src/auth/utils/AuthManager.jsx

```diff
@@ -0,0 +1,151 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+import AuthenticationAPI from '../../utils/apis/AuthenticationAPI';
+
+/**
+ * Name of the session cookie.
+ */
+const sessionUser = 'wso2dashboard_user';
+
+/**
+ * Authentication manager.
+ */
+export default class AuthManager {
+    /**
+     * Get user from the session cookie.
+     *
+     * @returns {{}|null} User object
+     */
+    static getUser() {
+        const buffer = AuthManager.getSessionCookie(sessionUser);
+        return buffer ? JSON.parse(buffer) : null;
+    }
+
+    /**
+     * Set user into a session cookie.
+     *
+     * @param {{}} user  User object
+     */
+    static setUser(user) {
+        AuthManager.setSessionCookie(sessionUser, JSON.stringify(user), 7 * 24 * 3600 * 1000);
+    }
```

Comment:
Don't we need to get this from a config rather than hard coding so that session expiry is configurable.

Reply:
+1. If that's so we need to introduce a new API to expose the configurations to client side from the deployment.yaml.

Reply:
Shall we handle this under separate issue since it need some major changes?

Reply:
Created the issue https://github.com/wso2/carbon-dashboards/issues/657 to handle above requirement.